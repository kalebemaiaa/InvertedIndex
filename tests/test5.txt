QueryOr(QueryAnd(QueryWord("deputado"), QueryWord("assessor")), QueryWord("ministro"))
2910.raw
642.raw
4513.raw
4107.raw
1277.raw
517.raw
5006.raw
2686.raw
3501.raw
2800.raw
1647.raw
11770.raw
4709.raw
4943.raw
1039.raw
680.raw
5411.raw
6229.raw
5556.raw
6414.raw
11461.raw
5618.raw
299.raw
146.raw
11985.raw
5813.raw
985.raw
5002.raw
3679.raw
3281.raw
11040.raw
3972.raw
1686.raw
858.raw
614.raw
11570.raw
3221.raw
11661.raw
5794.raw
3282.raw
4795.raw
3778.raw
3983.raw
4256.raw
2191.raw
926.raw
5447.raw
671.raw
3118.raw
4386.raw
5894.raw
5177.raw
676.raw
358.raw
300.raw
6008.raw
7794.raw
3900.raw
3142.raw
713.raw
11857.raw
5358.raw
6237.raw
3356.raw
3173.raw
1760.raw
5418.raw
1671.raw
6078.raw
4571.raw
1895.raw
5524.raw
5612.raw
3702.raw
3429.raw
4491.raw
1610.raw
1875.raw
3099.raw
11241.raw
5134.raw
1959.raw
1575.raw
1852.raw
2329.raw
2209.raw
683.raw
3843.raw
11265.raw
11701.raw
11796.raw
1551.raw
2679.raw
499.raw
1181.raw
5080.raw
1423.raw
11144.raw
2913.raw
2860.raw
2291.raw
3862.raw
3291.raw
2920.raw
1113.raw
1370.raw
6025.raw
2952.raw
2726.raw
77.raw
5554.raw
4172.raw
9010.raw
500.raw
10812.raw
6201.raw
6379.raw
5522.raw
4108.raw
266.raw
3373.raw
1041.raw
8895.raw
2446.raw
2053.raw
5432.raw
273.raw
6178.raw
673.raw
951.raw
6046.raw
2944.raw
383.raw
5386.raw
5290.raw
3888.raw
5782.raw
4315.raw
718.raw
1472.raw
3456.raw
1464.raw
1861.raw
3069.raw
3437.raw
10884.raw
2721.raw
2330.raw
4804.raw
4880.raw
577.raw
2836.raw
5212.raw
11901.raw
6082.raw
3301.raw
8718.raw
11616.raw
4458.raw
38.raw
10869.raw
7795.raw
276.raw
423.raw
4022.raw
4761.raw
10870.raw
11291.raw
461.raw
3014.raw
11680.raw
6277.raw
3260.raw
12029.raw
1768.raw
265.raw
2265.raw
1405.raw
3208.raw
2261.raw
2340.raw
5490.raw
6371.raw
1901.raw
542.raw
935.raw
6018.raw
2963.raw
1708.raw
604.raw
11274.raw
3323.raw
10810.raw
5399.raw
3838.raw
11054.raw
940.raw
2432.raw
15.raw
4665.raw
