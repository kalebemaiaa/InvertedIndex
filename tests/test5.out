47
325
395
626
734
742
754
887
918
940
999
1000
1245
1268
1311
1487
1524
1775
1806
1905
2031
2070
2092
2101
2164
2166
2227
2257
2272
2286
2323
2400
2538
2695
2851
2866
2890
3024
3035
3112
3213
3345
3597
3629
3693
3814
3830
4058
4145
4338
4446
4464
4764
4768
4790
4817
4883
4937
5012
5099
5178
5180
5308
5400
5905
5934
6215
6256
6258
6403
8758
8806
8877
10953
10999
11525
11882
